src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs

```diff
@@ -4,41 +4,161 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Globalization;
 using System.Threading.Tasks;
-using NuGet.Protocol.Core.Types;
 using NuGet.Protocol.VisualStudio;
 using NuGet.Versioning;
 
 namespace NuGet.PackageManagement.UI
 {
+    // This is the model class behind the package items in the infinite scroll list.
+    // Some of its properties, such as Latest Version, Status, are fetched on-demand in the background.
     public class SearchResultPackageMetadata : INotifyPropertyChanged
```

Comment:
This is a ViewModel, so it should be called something like SearchResultPackageViewModel